### DISCUSSION TITLE
problems with QTGMC on machine without graphics card

### DISCUSSION THREAD

**Discussion by bobdig** [staxrip/staxrip#1277] [Q&A] [answered]
I recently noticed a problem when I am trying to use QTGMC on my only Win11 machine. Staxrip  throws errors like this one. I did a fresh "install", doesn't help.
![Capture](https://github.com/staxrip/staxrip/assets/46788742/0a0d057b-7eca-49ac-8984-3f74f6907810)

No problem on my main machine with Win10, even fresh. So I have no clue what to do next.

**Comment by Dendraspis** [comment]
Can you post the log file or the script you have used?

**Reply by bobdig** [reply]
Sure, thx for looking.
[abc_temp.zip](https://github.com/staxrip/staxrip/files/14640958/abc_temp.zip)
Is probably not from the screenshot but another fail.

**Reply by Dendraspis** [reply, marked as answer]
Should be related to the used Graphics Driver. One of the dependencies seems to be GPU-accelerated, which is obviously not supported by your remote connection.

**Reply by bobdig** [reply]
Thanks! This machine only has Aspeed Graphics. I had an older geforce in it for a short time... but this probably isn't the problem here.

**Reply by jessielw** [reply]
You can use this to use the libraries without a GPU. Simply install this regardless if the cpu is AMD or intel

https://drive.google.com/file/d/1yqnG50khG-fLwWpjVUpVYwdUQjjGh7C1/view
